tools/ack-discover/controller.py

```diff
@@ -133,3 +157,70 @@ def get_runtime_and_aws_sdk_version(writer, repo, image_version):
         # proper Git tags for releases...
         pass
     return runtime_version, aws_sdk_version
+
+
+def get_github_issue(writer, gh, service):
+    """Returns the Github Issue for the service, or None if no such issue
+    exists.
+    """
+    global _sc_issues
+    ack_org = gh.get_organization("aws-controllers-k8s")
+    community_repo = ack_org.get_repo("community")
+    writer.debug(f"[controller.get_github_issue] finding Github Issue for {service.package_name} ...")
+    if _sc_issues is None:
+        sc_label = community_repo.get_label(name="Service Controller")
+        _sc_issues = community_repo.get_issues(labels=[sc_label])
+
+    for issue in _sc_issues:
+        # The GH issues with label "Service Controller" all have the same title
+        # pattern: "<Service Name> service controller"
```

Comment:
This feels like a big assumption. Will someone be going through and updating each of the service controller requests to match this?

Reply:
Not sure what else we can do, unfortunately... we can't really lock down the Issue title template :)